client/me/help/style.scss

```diff
@@ -46,3 +46,38 @@
 	font-size: 14px;
 	color: $gray-dark;
 }
+
+.help__course-teaser .card {
```

Comment:
@mtias The extra place is here:
![screen shot 2016-08-24 at 1 00 29 pm](https://cloud.githubusercontent.com/assets/1854440/17939959/dd723434-69fa-11e6-9ec8-e7ae5f2d5dd1.png)